tests/unit/EntityInfo.spec.ts

```diff
@@ -131,4 +134,104 @@ describe('EntityInfo', () => {
     expect(wrapper.find('#nr-subtitle').exists()).toBeFalsy()
     expect(wrapper.find('#nr-number').exists()).toBeFalsy()
   })
+
+  it('displays the breadcrumb correctly', async () => {
+    const store = getVuexStore()
+    const router = mockRouter.mock()
+
+    sessionStorage.clear()
+    sessionStorage.setItem('TEMP_REG_NUMBER', 'T123456789')
+
+    store.state.entityName = 'My Named Company'
+    const wrapper = mount(EntityInfo, { vuetify, store, router })
+
+    await Vue.nextTick()
+
+    const breadcrumbs = wrapper.vm.$el.querySelectorAll('.v-breadcrumbs li')
+    const crumb1 = breadcrumbs[0]
+    const divider = breadcrumbs[1] // Divider is present every odd index
+    const crumb2 = breadcrumbs[2]
+    const crumb3 = breadcrumbs[4]
+
+    expect(crumb1.textContent).toContain('Manage Businesses')
+    expect(divider.textContent).toContain('>')
+    expect(crumb2.textContent).toContain('My Named Company')
+    expect(crumb3).toBeUndefined()
+  })
+
+  it('displays the breadcrumb correctly when navigating to an annual report', async () => {
+    const store = getVuexStore()
+    const router = mockRouter.mock()
+    router.push('annual-report')
+
+    sessionStorage.clear()
+    sessionStorage.setItem('TEMP_REG_NUMBER', 'T123456789')
+
+    store.state.entityName = 'My Named Company'
+    store.state.ARFilingYear = '2020'
+    const wrapper = mount(EntityInfo, { vuetify, store, router })
+
+    await Vue.nextTick()
+
+    const breadcrumbs = wrapper.vm.$el.querySelectorAll('.v-breadcrumbs li')
```

Comment:
BTW, the other way to do this is with `wrapper.findAll()` and then you use `at()` to get each found item. No need to change atm.

Reply:
Interesting, thanks!